### ISSUE TITLE
A small feature request

### ISSUE THREAD

**Issue by brmbrmcar** [MaartenBaert/ssr#506]
I would like to be able to cancel the recording, or at least open the ssr window, with a keyboard shortcut. I know it doesn't seem like a big deal, but whenever I record a game in full screen and I need to restart it can take forever and if I need to do it multiple times it gets extremely frustrating.

**Comment by CarlosZM** [comment]
So, hotkey option which by default is ctrl+r (start/stop) record, is isn't enough?

**Comment by WLBI** [comment]
Yes you are right, a shortcut for start and stop recording would be a great thing!

**Comment by WLBI** [comment]
Isn't it already included? Strg+R to start/stop the recording

**Comment by brmbrmcar** [comment]
No, I mean actually canceling the recording.
